yarn-project/p2p/src/services/libp2p/libp2p_service.ts

```diff
@@ -622,9 +622,13 @@ export class LibP2PService<T extends P2PClientType> extends WithTracer implement
   public async propagate<T extends Gossipable>(message: T) {
     const p2pMessageIdentifier = await message.p2pMessageIdentifier();
     this.logger.trace(`Message ${p2pMessageIdentifier} queued`, { p2pMessageIdentifier });
-    void this.jobQueue.put(async () => {
-      await this.sendToPeers(message);
-    });
+    void this.jobQueue
+      .put(async () => {
+        await this.sendToPeers(message);
+      })
+      .catch(error => {
```

Comment:
added this catch 

race condition in test where a node would receive a block proposal, begin to process it and the test would end. Stopping the jobQueue, so when it got to propagating the response ( the attestation ) the job queue was stopped, it would throw, and the test would fail